docs/UserGuide.md

```diff
@@ -268,10 +268,15 @@ Format: `show-plan`
 ## Randomized Review
 #### Displaying a random Flashcard: `random`
 
-Shuffles and displays a random flashcard from the list of all flashcards.
+Shuffles and displays all the flashcards in a random order.
 
 Format: `random`
 
+The application will then prompt the user to give responses during the review:
+
+Format: `yes` to mark the flashcard just shown as reviewed
```

Comment:
Consider starting a new line for the 'no' response (we need a line break to start a new line in markdown)?